src/main/AndroidManifest.xml

```diff
@@ -31,7 +31,8 @@
     <uses-permission android:name="android.permission.MANAGE_ACCOUNTS" />
     <uses-permission android:name="android.permission.USE_CREDENTIALS" />
     <uses-permission android:name="android.permission.WAKE_LOCK" />
-    <uses-permission android:name="android.permission.WRITE_SETTINGS" />
+    <uses-permission android:name="android.permission.WRITE_SETTINGS"
```

Comment:
Ignoring linting warnings in this way might not be the best thing to do.
Ref: https://riptutorial.com/android/example/1135/using-tools-ignore-in-xml-files